content/how-to/add-code-servers/gitlab-cloud/add-token/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "How To generate a personal access token (PAT) in GitLab"
+description: "How To generate a personal access token (PAT) in GitLab"
```

Comment:
Sentence case.

If there is no description, it will be automatically generated based on the content. However, if we do provide one, it really should be different from the title.

Reply:
otoh, look at the difference between a the auto-generated and human-generated `description` here
![Screen Shot 2022-03-08 at 4 52 02 PM](https://user-images.githubusercontent.com/1709697/157351329-91a7b6b1-dc59-479b-a654-2f1466057f1a.png)
:

Reply:
I like the generated description, it gives a start to the article, I think this is the best option.